src/arch/xtensa/include/arch/task.h

```diff
@@ -73,7 +73,7 @@ struct irq_task **task_irq_high_get(void);
  * \brief Runs task.
  * \param[in,out] task Task data.
  */
-int arch_run_task(struct task *task);
+int arch_run_task(struct sof_task *task);
```

Comment:
struct task is fine, no need for sof_ prefix

Reply:
Done